google/cloud/speech/v1/cloud_speech_gapic.yaml

```diff
@@ -0,0 +1,52 @@
+type: com.google.api.codegen.ConfigProto
+generate_samples: true
+language_settings:
+  java:
+    package_name: com.google.cloud.speech.v1
+  python:
+    package_name: google.cloud.speech.v1
+  go:
+    package_name: google.golang.org/google/cloud/speech/v1
+  csharp:
+    package_name: Google.Cloud.Speech.V1
+  ruby:
+    package_name: Google::Cloud::Speech::V1
+  php:
+    package_name: Google\Cloud\Speech\V1
+interfaces:
+- name: google.cloud.speech.v1.Speech
+  collections: []
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 60000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 60000
+    total_timeout_millis: 600000
+  methods:
+  - name: Recognize
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    timeout_millis: 30000
```

Comment:
Already done in https://github.com/googleapis/toolkit/commit/9652cb012458e299e2077ad67e5472cba0dca2b3